identifiers/country-in/README

```diff
@@ -1,11 +1,14 @@
+
 README
```

Comment:
Could you add to the README a descritpion of the types being used within the `country-in` dir and what types of districts they map to?

Reply:
Added, PTAL